images/releng/ci/variants.yaml

```diff
@@ -1,11 +1,11 @@
 variants:
   go1.21-bookworm:
     CONFIG: 'go1.21.0-bookworm'
```

Comment:
Is this parameter supposed to include the patch release?

Reply:
that is wrong, fixed, thanks for the catch!